data/anaconda.conf

```diff
@@ -31,6 +31,15 @@ optional_modules =
     org.fedoraproject.Anaconda.Modules.Subscription
     org.fedoraproject.Anaconda.Addons.*
 
+# Save input kickstart to target system.
+save_input_kickstart = True
+
+# Save output kickstart to target system.
+save_output_kickstart = True
+
+# Save logs to target system.
+save_logs = True
```

Comment:
I am still not sure if the new options should be part of the `[Anaconda]` section or the `[Installation Target]` section.